theme/default/js/neon-custom.js

```diff
@@ -0,0 +1,2082 @@
+/**
+ *	Neon Main JavaScript File
+ *
+ *	Theme by: www.laborator.co
+ **/
```

Comment:
I think that my web designer using our caralog templates designer .


Em segunda-feira, 2 de fevereiro de 2015, Dan Garner <
notifications@github.com> escreveu:

> In theme/default/js/neon-custom.js
> <https://github.com/xibosignage/xibo-cms/pull/17#discussion_r23966250>:
>
> > @@ -0,0 +1,2082 @@
> > +/**
> > + *	Neon Main JavaScript File
> > + *
> > + *	Theme by: www.laborator.co
> > + **/
>
> What is the licence on this file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xibosignage/xibo-cms/pull/17/files#r23966250>.
>


-- 
*Laercio Junior*
Cel./Pessoal: (21) 99162-7440
Email Pessoal: laercionit@gmail.com
MSN: laercionit@hotmail.com
Skype: laercionit